manager/notifier.go

```diff
@@ -190,6 +191,15 @@ func (n *notifier) sendEmailNotification(email string, a *Alert) error {
 	return writeEmailBody(wc, a)
 }
 
+func (n *notifier) sendPushoverNotification(token, userKey string, a *Alert) error {
+	po, err := pushover.NewPushover(token, userKey)
+	if err != nil {
+		return err
+	}
+
+	return po.Message("Alertmanager " + a.Description)
```

Comment:
Actually, the client library does support setting titles:

http://godoc.org/github.com/thorduri/pushover#Pushover.Push

So how about:

```go
_, _, err := po.Push(&po.Message{
	Title: a.Summary,
	Message: a.Description,
})
return err
```

(there are potentially other interesting fields in http://godoc.org/github.com/thorduri/pushover#Message)